frontend/public/locales/en/apply.json

```diff
@@ -73,11 +73,11 @@
   "confirm": {
     "page-title": "Confirmation",
     "keep-copy": "Keep a copy of your application",
-    "print-copy-text": "You will <bold>not</bold> be receiving a <strong>confirmation email</strong>. It is important to keep a copy of your application. You can use this button to print.",
+    "print-copy-text": "You will <bold>not</bold> be receiving a <bold>confirmation email</bold>. It is important to keep a copy of your application. You can use this button to print.",
```

Comment:
`<b>` instead of `<bold>` since `bold` is not a valid HTML tag

Reply:
https://www.w3schools.com/html/html_formatting.asp#:~:text=HTML%20%3Cb%3E%20and%20%3Cstrong%3E%20Elements

Reply:
<bold> is not a literal HTML tag. Its a component place holder used with <Trans>
`<Trans ns={i18nNamespaces} i18nKey="confirm.print-copy-text" components={{ bold: <strong /> }} />`

Reply:
Gotcha. `strong` is supported by `Trans` component so you shouldn't have to do it this way. Look at `transKeepBasicHtmlNodesFor` config https://react.i18next.com/latest/trans-component.

Reply:
Oh! Very nice!